k8s-template/tinkerpop-graphdb-schema-loader.yaml

```diff
@@ -93,8 +126,8 @@ spec:
   ports:
     - name: http
       protocol: TCP
-      targetPort: ${GRAPHDB_PERSONA_SCHEMA_LOADER_PORT}
-      port: ${GRAPHDB_PERSONA_SCHEMA_LOADER_PORT}
+      targetPort: ${ENVOY_PORT}
+      port: 8182
```

Comment:
What's the rationale behind exposing graphdb on port 8182 as opposed to the standard graphdb port? In case the envoy proxy eventually needs to proxy more than one service? Is that likely at all?

Reply:
8182 _is_ the standard graphdb port, as can be seen here: https://github.com/UKHomeOffice/cdp-deployment-templates/blob/master/vars/common.cfg#L22

We could keep the variable in place if you like. It just didn't feel 'correct' to me. i.e. Changing the port that graphdb runs on and changing the port the service runs on are two different things; we don't need to couple them.

I also thought the explicit port number helped from a documentation / code clarity perspective.

Reply:
Should have double-checked my assumption. I thought the default graphdb port number was different. So no need to change anything